### PR TITLE
add three parameters to input.nml for NSSL MP

### PR DESCRIPTION
**Pull request by Jili-Dong**
To support RRFS multiphysics ensemble, three parameters in NSSL MP are being added to input.nml, including rain shape parameter (nssl_alphar), graupel-droplet collection efficiency (nssl_ehw0_in) and hail-droplet collection efficiency (nssl_ehlw0_in)

PR to ccpp/data/GFS_typedefs.F90/meta will be submitted to fv3atm shortly

Related to issue
#44

### CODE REVIEW

File: physics/mp_nssl.meta [hunk 1]
```diff
@@ -165,6 +165,30 @@
...
+[nssl_alphar]
+  standard_name = nssl_alpha_rain
+  long_name = rain PSD shape parameter in NSSL micro
+  units = none
+  dimensions = ()
+  type = real
+  kind = kind_phys
+  intent = in
+[nssl_ehw0_in]
+  standard_name = nssl_hw_collec_eff
```
Comment (grantfirl) [inline physics/mp_nssl.meta]: Please change the standard name to `nssl_graupel_collection_efficiency` here and in https://github.com/NOAA-EMC/fv3atm/pull/632

File: physics/mp_nssl.meta [hunk 2]
```diff
@@ -165,6 +165,30 @@
...
+  long_name = graupel droplet collection efficiency in NSSL micro
+  units = none
+  dimensions = ()
+  type = real
+  kind = kind_phys
+  intent = in
+[nssl_ehlw0_in]
+  standard_name = nssl_hlw_collec_eff
```
Comment (grantfirl) [inline physics/mp_nssl.meta]: Please change the standard name to `nssl_hail_collection_efficiency` here and in https://github.com/NOAA-EMC/fv3atm/pull/632

File: physics/mp_nssl.meta [hunk 3]
```diff
@@ -165,6 +165,30 @@
   type = real
   kind = kind_phys
   intent = in
+[nssl_alphar]
+  standard_name = nssl_alpha_rain
+  long_name = rain PSD shape parameter in NSSL micro
```
Comment (ChunxiZhang-NOAA) [inline physics/mp_nssl.meta]: Could you change 'micro' to 'microphysics scheme' and give a full name to 'PSD'?

Comment (Jili-Dong) [reply physics/mp_nssl.meta]: @grantfirl @ChunxiZhang-NOAA standard and long names are updated as requested. Other parameter names related to NSSL microphysics are also updated similarly for consistency. GFS_typedefs.meta also updated in fv3atm. Thanks for reviewing!

Comment (ChunxiZhang-NOAA) [reply physics/mp_nssl.meta]: Thank you Jili. This PR is in good shape now.

**Review by dustinswales — APPROVED** [review]
Everything looks good to me.
Thanks @grantfirl @ChunxiZhang-NOAA for the standard_name suggestions.

**Review by grantfirl — APPROVED** [review]

**Review by ChunxiZhang-NOAA — APPROVED** [review]

**Comment by MicroTed** [comment]
It is fine with me to add these. I would suggest dropping the "\_in" part of the names, though, since the loose convention is to use "nssl_[name of parameter in the scheme]" 

The alternative way to access these parameters is through a local namelist (nssl_mp_params) that can be read by the init routine. But I don't know if there is a policy/guideline about doing that within CCPP? For a "research" version, it might be good to document this option.

**Pull request merged**